components/molecule/photoUploader/src/PhotosPreview.js

```diff
@@ -179,7 +198,7 @@ const PhotosPreview = ({
       setList={setFiles}
       animation={200}
       draggable={`.${THUMB_SORTABLE_CLASS_NAME}`}
-      onEnd={() => _onSortEnd()}
+      onEnd={sortedFile => _onSortEnd(sortedFile)}
```

Comment:
Just for consistency with _onSortEnd func argument naming
```suggestion
      onEnd={event => _onSortEnd(event)}
```